kazoo/handlers/utils.py

```diff
@@ -291,12 +305,55 @@ def wrap(async_result):
     :param async_result: An async result implementing :class:`IAsyncResult`
 
     """
+
     def capture(function):
         @capture_exceptions(async_result)
         def captured_function(*args, **kwargs):
             value = function(*args, **kwargs)
             if value is not None:
                 async_result.set(value)
             return value
+
         return captured_function
+
     return capture
+
+
+def _selector_select(rlist, wlist, xlist, timeout=None, *,
```

Comment:
You dropped `xlist` & `xevents` in this function. Was this intentional?

Reply:
Because the selectors module didn't implement the support of xevent, the same as gevent/eventlet.